recipes/hiredis/0.14.1/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+
+from os import rename
+from os.path import join
```

Comment:
Just import `os` and use `os.path.join` and `os.rename`.
Importing these makes the code harder to understand.

```suggestion
import os
```